Azure/sonic-buildimage
# [device]:Fixed as5835_54x port mapping issue

Signed-off-by: derek_sun <derek_sun@edge-core.com>

**- What I did**
Fix 100G port mapping.

**- How I did it**
modify port_config.ini about 100G port mapping.

**- How to verify it**
Upload port_config.ini into switch, use show interface show to confirm port link-up/down.

**- Description for the changelog**
Fixed as5835_54x port mapping issue.
